### PR TITLE
KAFKA-5619: Make --new-consumer option as deprecated in all tools

### PR DESCRIPTION
**Pull request by ppatierno**
Added deprecation and warning message on --new-consumer option usage with ConsumerGroupCommand and ConsumerPerformance tools.

### CODE REVIEW

**Comment by omkreddy** [comment]
+1

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/6196/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6212/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by ppatierno** [comment]
@ijuma do you think that it's ok for merging like the other PR #3537  strictly related to ConsoleConsumer ?

File: core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala [hunk 1]
```diff
@@ -709,7 +709,7 @@ object ConsumerGroupCommand extends Logging {
       "Pass in just a topic to delete the given topic's partition offsets and ownership information " +
       "for every consumer group. For instance --topic t1" + nl +
       "WARNING: Group deletion only works for old ZK-based consumer groups, and one has to use it carefully to only delete groups that are not active."
-    val NewConsumerDoc = "Use new consumer. This option requires that the 'bootstrap-server' option is used."
+    val NewConsumerDoc = "This is the default, so this option is deprecated and will be removed in a future release."
```
Comment (ijuma) [inline core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala:712]: We should still describe the config's purpose.

File: core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala [hunk 2]
```diff
@@ -808,6 +808,11 @@ object ConsumerGroupCommand extends Logging {
       } else {
         CommandLineUtils.checkRequiredArgs(parser, options, bootstrapServerOpt)
 
+        if (options.has(newConsumerOpt)) {
+          Console.err.println("The --new-consumer option is deprecated and will be removed in a future major release." +
+            "The new consumer is used by default if the --bootstrap-server option is provided.")
```
Comment (ijuma) [inline core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala:813]: We should still describe the config's purpose.

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM aside from a couple of minor remarks. I'll fix them both before merging to trunk.

**Pull request closed without merging**